textalyze.js

```diff
@@ -17,7 +18,26 @@ function itemCounts(array) {
   // Your code here.
   // Run 'npm test' to see what tests need to pass.
 
+  array.forEach((value) => {
+    const sum = counts.get(value) ? counts.get(value) + 1 : 1;
+    counts.set(value, sum);
+  });
+
   return counts;
 }
+const array = ['car', 'house', 'car', 'b', 'c', 'd', 'c'];
+result = itemCounts(array);
+console.log(result);
+
+/**
+ * Takes an arbitrary String as input
+ * and returns an Array with the counting of all the characters in the string
+ * @param {String} arbitrary string
+ * @returns {Array} a Map with the counting of each char
+ */
+function stringCharsToArray(string) {
+  const arrayString = string.split('');
+  return itemCounts(arrayString);
```

Comment:
The function you implemented isn't what you've described in the comments.  This returns a `Map` of counts, not the array of things-to-be-counted.

There's no reason for `stringCharsToArray` to know about `itemCounts`.  The code can compose the two functions if that's called for.

So, the comment is the way to go and the behavior of the function should be changed.